app/views/partials/personal_key/_entry_fields.html.slim

```diff
@@ -1,13 +1,9 @@
 .mb3
-  label.block.bold#personal-key-label
-    = t('simple_form.required.html') + t('forms.two_factor.personal_key')
-  .clearfix.mxn1.sm-mxn2.separator-inputs
-    - Figaro.env.recovery_code_length.to_i.times do
-      .col.col-3.px1.sm-px2
-        = f.input attribute_name, as: :array, label: false,
-          input_html: { 'aria-labelledby' => 'personal-key-label',
-            autocapitalize: 'none',
-            autocomplete: 'off',
-            maxlength: RandomPhrase::WORD_LENGTH,
-            required: true,
-            spellcheck: 'false' }
+  = f.input attribute_name,
+    label: t('simple_form.required.html') + t('forms.two_factor.personal_key'),
+    input_html: { autocapitalize: 'none',
+      autocomplete: 'off',
+      class: 'pkey caps',
+      maxlength: (RandomPhrase::WORD_LENGTH * Figaro.env.recovery_code_length.to_i + 3),
```

Comment:
the `+3` could be `(Figaro.env.recovery_code_length.to_i - 1)` to avoid hardcoding a magic number

But maybe we don't even need maxlength here? It would be fine without it?